project_forecast_line_deadline/models/project_task.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2024 Therp BV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+
+
+class ProjectTask(models.Model):
+    _inherit = "project.task"
+
+    # Make forecast end date computable and stored
+    forecast_date_planned_end = fields.Date(
+        compute="_compute_forecast_date_planned_end", inverse=lambda x: None, store=True
```

Comment:
I haven't seen this `inverse=lambda x: None` construction before.

What does it mean in practice?

My guess from looking at the code would be:
- It's possible to write to this computed field
- However, writing to it, just clears it
- Subsequently changing the deadline on the task, again recomputes this field to be the same as the deadline field

If that's how it is and it's intended behaviour, this is fine by me, but just doublechecking. It might also be good to explain it with a comment note.